whois-api/src/main/java/net/ripe/db/whois/api/rest/RestServiceHelper.java

```diff
@@ -173,6 +175,11 @@ public static WebApplicationException createWebApplicationException(final Runtim
         return new WebApplicationException(responseBuilder.build());
     }
 
+    public static boolean isHttpProtocol(final HttpServletRequest request) {
+        final String header = request.getHeader(HttpHeader.X_FORWARDED_PROTO.toString());
+        return !org.apache.commons.lang.StringUtils.isEmpty(header) && HttpScheme.HTTP.is(header);
+    }
```

Comment:
Can you put import statement instead of putting fully qualified name here?

Reply:
Done